Source/Device.swift.gyb

```diff
@@ -363,6 +364,21 @@ public enum Device {
   public var localizedModel: String {
     return UIDevice.current.localizedModel
   }
+
+  /// PPI (Pixels per Inch) on the current device's screen (if applicable). When not applicable the property returns nil.
+  public var ppi: Int? {
+    #if os(iOS)
+    switch self {
+      % for device in iOSDevices:
+      case .${device.caseName}: return ${device.ppi}
+      % end
+      default:
```

Comment:
@dennisweissmann @giacgbj now that you mention it, it would make sense. I guess i was just being a nazi about the Simulator's ppi actually depending on your Mac's display resolution. In the real application of a Simulator though, i guess the point would be to, well, emulate the real device for development purposes.

Sure guys, i can make the change.